sdk-extension/opentelemetry-sdk-extension-aws/src/opentelemetry/sdk/extension/aws/resource/__init__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Nit: `opentelemetry.sdk.extension.aws.resources` plural to match `opentelemetry.sdk.resources`? Not a big deal though.

Reply:
Oh I didn't even notice that, actually I did `resource` because that is the folder it is in [on the specifications](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource)

Also in [java we put it under resource](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/aws/src/main/java/io/opentelemetry/sdk/extension/aws/resource)

And yet [JavaScript](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-resource-detector-aws/src/detectors), [Go](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/detectors) and [PHP](https://github.com/open-telemetry/opentelemetry-php-contrib/tree/main/detectors) all put it under a folder called `detectors/` 😅 

I guess I'll defer to the SIG here, as GCP and other vendors will probably add their resource Detectors soon... maybe we should even move the SDK import path? 😓 Although I imagine that will be hard after 1.0.

Reply:
We aren't at `1.0` yet so nows the time to decide on an import path :D. But again, not a blocker.

Reply:
Ah, I meant we should move `opentelemetry.sdk.resources` to `opentelemetry.sdk.resource` to match the spec 🙂

I actually like `.resource` because there is only 1 `resource:` namespace in the attributes. Even if you add multiple `ResourceDetector`s you're still setting attributes in 1 namespace...

I'll leave it for now and if we need to we'll change it before this package goes 1.0 😄. Shouldn't be a big deal to do "both" if we need to as well later!